Tutorials/src/mcHistograms.cxx

```diff
@@ -33,6 +35,26 @@ struct VertexDistribution {
   }
 };
 
+// Simple analysis of PhysicalPrimary particles
+struct PhysicalPrimaryCharge {
+  OutputObj<TH1F> charge{TH1F("charge_prim", "charge_prim", 100, -5, 5)};
+  TDatabasePDG pdgDB;
+
+  void process(aod::McParticles const& mcParticles)
+  {
+    for (auto& particle : mcParticles) {
+      if (!particle.isPhysicalPrimary()) {
+        continue;
+      }
+      auto pdgParticle = pdgDB.GetParticle(particle.pdgCode());
+      if (!pdgParticle) {
+        continue;
+      }
+      charge->Fill(pdgParticle->Charge());
```

Comment:
@aalkin @ktf @njacazio I remember some more efficient way to get the charge. Am I mistaken?

Reply:
@sawenzel there is a slightly better way to use the PDG database, see here:
https://github.com/AliceO2Group/O2Physics/blob/master/Tutorials/src/usingPDGService.cxx#L20 
Most importantly, note that the charge in PDG is in units of `|e|/3`
```cpp
 Double_t         fCharge;           // charge in units of |e|/3
 ```

@jgrosseo unfortunately, no, we still need to query PDG.